src/collapse/collapse.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, Input} from 'angular2/angular2';
+
+@Directive({
+  selector: '[ngb-collapse]',
+  host: {'class': 'collapse', '[class.in]': '_open', '[attr.aria-expanded]': '_open', '[style.height.px]': '_height'}
+})
+export class NgbCollapse {
+  private _open: boolean;
+  private _height: number;
```

Comment:
Since this goes into the template, it shouldn't be private nor have a underscore.